charger/mcc.go

```diff
@@ -202,8 +202,14 @@ func (mcc *MobileConnect) getEscapedJSON(uri string, result interface{}) error {
 	b, err := mcc.Request(req)
 	if err == nil {
 		var s string
-		if s, err = strconv.Unquote(strings.Trim(string(b), "\n")); err == nil {
-			err = json.Unmarshal([]byte(s), &result)
+		// ""\n is usually returned if no data is present, e.g. after a restart
+		// we know this is no valid json so we make sure we don't treat this as an error
+		if string(b) == "\"\"\n" {
+			return nil
+		} else {
+			if s, err = strconv.Unquote(strings.Trim(string(b), "\n")); err == nil {
+				err = json.Unmarshal([]byte(s), &result)
```

Comment:
what happens to the final err?

Reply:
Du kannst das unquote auch rausziehen. Welchem Zweck dient das letzte `err=Unmarshal` wenn das Resultat verworfen wird? Ich verstehe nicht warum der Linter oder Compiler da nicht meckert?

Reply:
da fängt er JSON Parsing fehler ab. Genau das ist ja passiert wenn keine Ladung seit dem Start der Box da war